### DISCUSSION TITLE
集群调度 - DaoCloud Enterprise

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#909] [Announcements]
# 集群调度 - DaoCloud Enterprise

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/kpanda/07UserGuide/Workloads/PodConfig/SchedulingPolicy/

**Comment by liyuerich** [comment]
工作负载反和性 - 应该是 工作负载反亲和性

**Reply by Michelle951** [reply]
fixed
